src/etld.lisp

```diff
@@ -32,8 +32,10 @@
          finally (return (list normal-tlds wildcard-tlds special-tlds))))))
 
 (defvar *etlds*
-  #-abcl '#.(load-etld-data)
-  #+abcl (load-etld-data))
+  (or
+    #+(and ecl win32 msvc) (load-etld-data)
+    #+abcl (load-etld-data)
```

Comment:
Merge the above 2 lines to use the same condition as the line below, but with `+`:

```lisp
#+(or abcl (and ecl win32 msvc))
```